src/core/AudioEngine.cpp

```diff
@@ -134,15 +133,11 @@ AudioEngine::AudioEngine( bool renderOnly ) :
 	// allocte the FIFO from the determined size
 	m_fifo = new Fifo( fifoSize );
 
-	// now that framesPerPeriod is fixed initialize global BufferManager
-	BufferManager::init( m_framesPerPeriod );
+	m_outputBufferRead = new std::array<surroundSampleFrame, m_framesPerPeriod>;
+	m_outputBufferWrite = new std::array<surroundSampleFrame, m_framesPerPeriod>;
 
-	int outputBufferSize = m_framesPerPeriod * sizeof(surroundSampleFrame);
-	m_outputBufferRead = static_cast<surroundSampleFrame *>(MemoryHelper::alignedMalloc(outputBufferSize));
-	m_outputBufferWrite = static_cast<surroundSampleFrame *>(MemoryHelper::alignedMalloc(outputBufferSize));
-
-	BufferManager::clear(m_outputBufferRead, m_framesPerPeriod);
-	BufferManager::clear(m_outputBufferWrite, m_framesPerPeriod);
+	std::fill(m_outputBufferRead->begin(), m_outputBufferRead->end(), 0);
+	std::fill(m_outputBufferWrite->begin(), m_outputBufferWrite->end(), 0);
```

Comment:
Since the array of `surroundSampleFrame`s would be value initialized if you were to use `std::make_unique`, this can be removed.
```suggestion
```